provider/core_route_table_resource_test.go

```diff
@@ -14,13 +14,24 @@ import (
 
 type ResourceCoreRouteTableTestSuite struct {
 	suite.Suite
-	Client       *baremetal.Client
-	Provider     terraform.ResourceProvider
-	Providers    map[string]terraform.ResourceProvider
-	Config       string
-	ResourceName string
+	Client              *baremetal.Client
+	Provider            terraform.ResourceProvider
+	Providers           map[string]terraform.ResourceProvider
+	Config              string
+	ResourceName        string
+	DefaultResourceName string
 }
 
+var defaultRouteTable = `
+resource "oci_core_route_table" "default" {
+	manage_default_resource_id = "${oci_core_virtual_network.t.default_route_table_id}"
```

Comment:
what is the expected behavior if user removes the manage default resource id after creating the resources

Reply:
This should fail with the same error as above.